backend/utils/message_utils.py

```diff
@@ -31,9 +31,8 @@ async def get_room_messages(org_id: str, room_id: str) -> list:
 
     DB = DataStorage(org_id)
     response = await DB.read(MESSAGE_COLLECTION, query={"room_id": room_id})
```

Comment:
Could you add an option for sorting the returned messages?

Reply:
@pauline-banye. Yeah, let's see first what the FE devs would like to add.